build-pipelines-PR.yml

```diff
@@ -129,7 +93,7 @@ jobs:
     enabled: True
     inputs:
       solution: src/client.sln
-      msbuildArgs: -p:Localize=true -p:ConsumeEndpointManager=false
```

Comment:
why remove this part? This taks is the one failing when PR is sent from fork, is this the reason why?

Reply:
so on the other parameter ConsumeEndpointManger is false because we are building/publishing endpoint manager as separate step in the pipelines so that main dsc project doesn't do it again. By default value is true. 

![image](https://user-images.githubusercontent.com/105889062/224369162-3c76e66d-2a86-48a7-8d83-48ed8f828c44.png)

Reply:
The failures for the forked repos are related to the external endpoints being present in the restore endpoint manager step above https://github.com/Azure/Bridge-To-Kubernetes/pull/193/files#r1131937016. That's why this step is failing, please see below screenshot. 

![image](https://user-images.githubusercontent.com/105889062/224369724-bc9779f0-5882-43d0-8557-e22004920cc2.png)

![image](https://user-images.githubusercontent.com/105889062/224370167-07c534db-95d1-41ef-8ca1-344bbd76c4a8.png)